src/reducers/loginReducer.js

```diff
@@ -2,12 +2,12 @@ import * as types from "../actions/actionTypes.js";
 
 const initialState_login = {
   token: "",
-  loginSuccess: null,
+  loginSuccess: true,
```

Comment:
I was wondering, shouldn't it be false at first... When the request is successful, then it becomes true?

Reply:
the loginSuccess

Reply:
It's just used to indicate whether I need to display a "Wrong username or password" message, so true is good for this usecase.

The functionality you are referring to is basically done by the toekn

Reply:
Oh ok then... looks good